help/policies/_posts/2024-02-12-mirrors.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: "ADS Mirror Links Update"
+order: 5
+category: policies
+---
+
+Previously, ADS had mirror sites hosted at servers across the world that were listed at [http://adsabs.harvard.edu/mirrors.html](http://adsabs.harvard.edu/mirrors.html). However, these mirror sites have been defunct since mid-2018 when we upgraded our infrastructure from being hosted at local servers to cloud based. This move to the cloud was made to improve the speed and reliability of ADS as our content base was growing. For more information, please see this blog post outlining the differences between the [Classic ADS and the New ADS](https://ui.adsabs.harvard.edu/blog/technical). 
```

Comment:
I'd reword this to seem less like an update and more like a policy. Maybe something like: "ADS does not currently host mirror sites, unlike ADS Classic. We removed our mirror sites in 2018 when we upgraded our site infrastructure from being hosted on local servers to cloud based... [keep the rest as-is]"